Project-II-Calculator/calculator/src/components/DisplayComponents/MainRows.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Display.css';
+import ActionButton from '../ButtonComponents/ActionButton'
+import NumberButton from '../ButtonComponents/NumberButton'
+
+const numbers = [1,2,3,4,5,6,7,8,9]
+
+const MainRows = function() {
+
+  return (
+    <div className='main-rows'>
+      <ActionButton text='clear' />
+      <div className='number-icons'>
+        {numbers.map(function(number) {
+          return <NumberButton text={number} buttonStyles='fit-width'/>
```

Comment:
To get rid of console log warning about `Each child in a list should have a unique "key" prop.`, you can add a `key` props to `NumberButton` component and set the value to the index of the array